packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2015,7 +2015,9 @@ async function handler(
         workflows: eventType.workflows,
         smsReminderNumber: smsReminderNumber || null,
         calendarEvent: { ...evt, ...{ metadata, eventType: { slug: eventType.slug } } },
-        isNotConfirmed: evt.requiresConfirmation || false,
+        isNotConfirmed: rescheduleUid
+          ? originalRescheduledBooking?.status !== BookingStatus.ACCEPTED
```

Comment:
I think we should make it `false` in case of a reschedule

Even when the booking is not yet confirmed we want the reschedule workflow to trigger, let me know if you think differently about that 🙏

Reply:
Yeah that works too I will make the changes.